tests/scr-readingsystem-features/OPS/content_001.xhtml

```diff
@@ -0,0 +1,88 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>epubReadingSystem object features</title>
+    <script>
+      const required_features = [
+        'dom-manipulation', 
+        'layout-changes', 
+        'touch-events', 
+        'mouse-events',
+        'keyboard-events',
+        'spine-scripting'
+       ]
+
+      /**
+       * A simple change that proves that scripting is enabled.
+       * 
+       * Note that the script is more complicated than would be necessary in HTML, because the innerHTML method does not
+       * seem to work in xhtml (only in html) :-(
+       */ 
+      function test_ers_features() {
+        const support = document.getElementById('scripting_support');
+        support.textContent = 'This Reading System supports scripting.';
```

Comment:
Nit: we usually don't capitalize Reading System. (For example, you did not in your description tags.) Applies throughout PR, but optional.

Reply:
+1

Reply:
I also made the changes in the other tests